### ISSUE TITLE
Update NPM Package

### ISSUE THREAD

**Issue by cjbuchmann-work** [c3js/c3#2029]
Hey all, any plans to update the npm package in the near future? We'd like to make use of the some of the new features that have been added since 0.4.11, which is just a hair over a year old now.

In particular, I'm hoping to make use of this https://github.com/c3js/c3/pull/1813

Thanks!

**Comment by netil** [comment]
C3.js wasn't maintained for a while.
Take in consideration of #1965.

**Comment by cjbuchmann-work** [comment]
Thanks @netil, I hadn't noticed that.

@masayuki0812 @aendrew, any thoughts here? If there's anything that I can do to help move this along, please let me know.

**Comment by cjbuchmann-work** [comment]
@masayuki0812 @aendrew any updates?
@netil what are your thoughts on this? I'm debating just creating a new npm package, but I'd really rather just push an update to the existing one if possible.

**Comment by netil** [comment]
@cjbuchmann-work 
Currently I'm working a forked new library, which will have same interface with D3 v4 support.

I don't know exactly what you've been added (new tooltip_order option, if i'm not wrong?), but it can be implemented via PR after release of that.

If works done as scheduled, I'm planning release within June :)

**Comment by MythicAngel** [comment]
Ping: @kt3k 

Please include @netil as your collaborator. It is refreshing to see that someone is committed to this project now. Also, if you need help, then I can help to flag the issues so that we can get a good overview about this library's current state.

**Comment by kt3k** [comment]
Yup, I joined maintainer group lately and have `write` access to the repo. I don't have `admin` access yet and therefore can't add @netil right now, but I can talk about this to @masayuki0812 later.
I'm planning to have a meeting with him in person about every month.

**Comment by netil** [comment]
Actually I don't know what's going on this project. I tried to contact contributors several months ago for any contribution opportunity, and I couldn't receive any positive feedback. That's why I decided to start new project.

I noticed that with #2033,  new commiter(@kt3k) was added to continue v1, but I'm almost finalizing the project to public release. If you needs some talks, plz contact me by mail.
